### PR TITLE
Cleanup deprecations.

### PR DESCRIPTION
**Pull request by simoalx**
Use more appropriate syntax for renamed syntax.

### CODE REVIEW

**Comment by JoeMatt** [comment]
Does `deprecated: 0.1.4` do anything outside of Apple APIs?

> These arguments begin with one of the following platform or language names:
> 
> iOS
> iOSApplicationExtension
> macOS
> macOSApplicationExtension
> watchOS
> watchOSApplicationExtension
> tvOS
> tvOSApplicationExtension
> swift
[via](https://docs.swift.org/swift-book/ReferenceManual/Attributes.html#//apple_ref/doc/uid/TP40014097-CH35-ID347)

Changing `*` to `swift` is telling XCode it's deprecated in Swift 0.1.4 essentially if I'm not misunderstanding Apple's docs.

A limitation I wish they would allow framework vendors with semantic versioning to use as well.

**Comment by simoalx** [comment]
Yeah, you're totally right - I misunderstood how things worked! Fixing up now

**Comment by simoalx** [comment]
Removed the version numbers - you're totally right, it would be so nice for the system to integrate with the semantic versioning of your project.

File: Sources/Extensions/UIViewController+Hero.swift [hunk 1]
```diff
@@ -332,49 +332,49 @@ public extension HeroExtension where Base: UIViewController {
...
 
 extension UIViewController {
-  @available(*, deprecated: 0.1.4, message: "use hero.dismissViewController instead")
+  @available(*, deprecated, renamed: "hero.dismissViewController()")
   @IBAction public func ht_dismiss(_ sender: UIView) {
     hero.dismissViewController()
   }
 
-  @available(*, deprecated: 0.1.4, message: "use hero.replaceViewController(with:) instead")
+  @available(swift, deprecated, renamed: "hero.replaceViewController(with:)")
```
Comment (JoeMatt) [inline Sources/Extensions/UIViewController+Hero.swift]:
```suggestion
  @available(*, deprecated, renamed: "hero.replaceViewController(with:)")
```

**Comment by simoalx** [comment]
Squashed suggestion into previous commit.

**Pull request merged**